.github/workflows/update-sysdig-deploy-chart.yaml

```diff
@@ -1,4 +1,4 @@
-name: update_sysdig_deploy_chart_versions
+name: Update the Sysdig deploy chart versions
```

Comment:
Can we please change `Sysdig deploy` to `sysdig-deploy` here, since it's the actual chart name?

Reply:
sounds good on it.